### ISSUE TITLE
Błąd dwóch prowadzących - zajęcia tego samego dnia o tej samej godzinie

### ISSUE THREAD

**Issue by tomertin** [pjatk21/alt-api#123]
**Czy błąd jest krytyczny (uniemożliwia korzystanie z jakiejś części API)**
Teoretico tak - widzisz tylko jednego z 2 prowadzących danego dnia (podczas tej godziny)

**Kroki do odtworzenia problemu**
Wybierz 2 prowadzących mających zajęcia o tej samej godzinie.
![problem](https://user-images.githubusercontent.com/88508650/173237512-2aa04c37-8e68-45eb-bca7-bd311d520c6b.png)

**Dodatkowe informacje (środowisko, języki programowania, narzędzia itp.)**
comic sans goes brr

**Comment by rafalopilowski1** [comment]
Poprawka na szybko: ograniczyć widok prowadzącego do 1 osoby

Na później: dla każdej osoby (student, wykładowca) osobna oś czasu?

**Comment by kpostekk** [comment]
Yup, nie implementowałem opcji, gdzie uwzględnia się dwa przedmioty w tym samym czasie. Zostawiam póki co jako open, będzie to priority w kolejnym update.

**Comment by rafalopilowski1** [comment]
Poprawka na szybko zaimplementowana - zostawiam otwarte, bo fajnie by było mieść 2 wykładowców na raz.
